controllers/limited_support/limited_support_controller.go

```diff
@@ -48,6 +51,12 @@ func (r *LimitedSupportConfigMapReconciler) Reconcile(ctx context.Context, req c
 	reqLogger := log.WithValues("Request.Namespace", req.Namespace, "Request.Name", req.Name)
 	reqLogger.Info("Reconciling Limited Support ConfigMap")
 
+	uuid := os.Getenv(EnvClusterID)
+	if uuid == "" {
+		return ctrl.Result{}, fmt.Errorf("cluster ID returned as empty string")
+	}
+	r.MetricsAggregator.SetClusterID(uuid)
```

Comment:
This is setting a metric that's already set in another controller. This needs to be removed